moveit_kinematics/ikfast_kinematics_plugin/templates/ikfast61_moveit_plugin_template.cpp

```diff
@@ -136,8 +136,12 @@ enum IkParameterizationType
   /// when serializing the ik parameterizations
 };
 
+// HACK to fix isnan ambiguity
```

Comment:
While it is sort of a hack, it would be nice if we could call this a *work around*, as it works around what is currently a limitation of OpenRAVE/IKFast.